src/openxr/OpenXRApi.h

```diff
@@ -285,6 +284,13 @@ class OpenXRApi {
 	XrSystemId get_system_id() { return systemId; }
 	XrSpace get_play_space() { return play_space; }
 	XrFrameState get_frame_state() { return frameState; }
+	String get_system_name() const { return system_name; }
+	uint32_t get_vendor_id() const { return vendor_id; }
+
+	bool set_render_target_size_multiplier(float multiplier);
```

Comment:
I think we should also add a `get_render_target_size_multiplier` and expose this as a property on our `OpenXRConfig` object, it's been a much requested feature.

Reply:
Done!
I'm only adding the core logic in this PR. I'll add the matching gdclasses properties and methods in a separate PR.